third_party/Makefile

```diff
@@ -25,8 +25,7 @@ endif
 # The version we use is the released tag of cuda-samples which is consistent with the cuda version in the environment or docker.
 # The Makefile of bandwidthTest does not have 'install' target, so need to copy bin to $(SB_MICRO_PATH)/bin/ and create $(SB_MICRO_PATH)/bin/ if not existing.
 bandwidthTest: sb_micro_path
-ifneq (,$(wildcard cuda-samples/Samples/bandwidthTest/Makefile))
-	cd cuda-samples && git checkout v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2)
+	if [ -d cuda-samples ]; then rm -rf cuda-samples; fi
+	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
```

Comment:
If you don't use git submodule, maybe you can clone through svn so that you can get the `bandwidthTest` subdirectory directly.
```suggestion
	svn checkout https://github.com/NVIDIA/cuda-samples/tags/v$$(nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2)/Samples/bandwidthTest
```

Reply:
![image](https://user-images.githubusercontent.com/37182275/125751495-d946b575-6d34-4931-b56e-b72c0f5ff59a.png)
Only clone bandwidthTest can not work, it seems it needs cuda-samples/Common/helper_cuda.h